src/StatsdClient/Statsd.cs

```diff
@@ -50,7 +50,22 @@ public abstract class Metric : ICommandType
             }
         }
 
-        public class Event : ICommandType
+        public abstract class Command : ICommandType
+        {
+            protected static string EscapeContent(string content)
+            {
+                return content
+                    .Replace("\r", "")
+                    .Replace("\n", "\\n");
+            }
+
+            protected static string TruncateOverage(string str, int overage)
+            {
+                return str.Substring(0, str.Length - overage);
+            }
+        }
+

```

Comment:
I am wondering if having these two methods defined `private` in the `Statsd` class is not enough. In particular, I found counter-intuitive that `Event`, `ServiceCheck` inherit from `Command` but not `Metric`. What do you think ?